client/src/components/App.jsx

```diff
@@ -22,6 +24,13 @@ background-color: #ffffff;
 font-family: "Times New Roman", Times, serif;
```

Comment:
is there times new roman in this widget?

Reply:
Yes there is. I was surprised